zerver/tests/test_invite.py

```diff
@@ -2301,6 +2305,7 @@ def test_successful_get_open_invitations(self) -> None:
         self.assertEqual(invites[0]["email"], "TestOne@zulip.com")
         self.assertTrue(invites[1]["is_multiuse"])
         self.assertEqual(invites[1]["invited_by_user_id"], hamlet.id)
+        self.assertEqual(set(invites[0]["stream_ids"]), set(stream_ids))
```

Comment:
Would be good to check this field for multiuse invites and also check "include_realm_default_subscriptions" field.